tensorflow/lite/kernels/transpose_test.cc

```diff
@@ -372,6 +405,19 @@ TEST(TransposeTest, Test4DFlattenTwo) {
                                                13, 10, 14, 11, 15}));
 }
 
+TEST(TransposeTest, Int8MismatchedQuantizationFails) {
+  // Input and output have different scale/zero_point.
+  // Construct model with INT8 input/output and then override the output
+  // quantization params to create a mismatch.
```

Comment:
The comments in the tests make the test harder to read actually. Please remove all comments that don't bring anything useful, but were added by an AI (both in the tests and in the code)